### ISSUE TITLE
Are you using the CryptoTrader for real-market trades?

### ISSUE THREAD

**Issue by petre-c** [gaat21/CryptoTrader#2]
Do you have some stats for the success rate?

**Comment by gaat21** [comment]
Yes, I had traded in realtime on the Binance and Poloniex, but my strategy was not realy good that's why I stopped the real-time trading after few hours. 
Success rate is depends on your startegy.

**Comment by 1ordRadium** [comment]
First of all thank you gaat21 for the project and the code! 

Your code seemed to me as the most advanced crypto project for C# on github. That's why I took it as a basis for my trading bot. It is currently live trading on Poloniex (5 minute time frame).

I had to rewrite some code to get the project running. One of my requirements was visual backtesting. I want to see what happens there ;-)
For that I had to transfer the TradingTester and the TradingCosole into a .Net 4.6.1 project. That was a bit painful. (i never worked before with .Net Core stuff)

I also converted the database part to SQLite and made a few other changes. The last one was better protection in case of network failure (improve your PoloniexExchangeProvider).

A big problem seems to be the API query at higher time frames. Poloniex does not deliver candels for 5 minutes fast enough. Next step is to push api subscription? ;)

Sorry i'm new to github, so i must get the grip first ;D
Keep up the nice work!
